scripts/dev/setup_kind_cluster.sh

```diff
@@ -29,3 +29,17 @@ containerdConfigPatches:
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
     endpoint = ["http://${ip}:${reg_port}"]
 EOF
+
+# Document the local registry
+# https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/generic/1755-communicating-a-local-registry
+cat <<EOF | kubectl apply -f -
```

Comment:
without this `ConfigMap` the local registry doesn't work properly and the Operator pod has problems
```
  Warning  Failed     4m19s (x4 over 5m49s)  kubelet            Failed to pull image "localhost:5000/mongodb-kubernetes-operator:latest": rpc error: code = Unknown desc = Error response from daemon: Get http://localhost:5000/v2/: dial tcp 127.0.0.1:5000: connect: connection refused
```